docs/en/stack/ml/df-analytics/flightdata-classification.asciidoc

```diff
@@ -351,15 +350,16 @@ The snippet below shows a part of a document with the annotated results:
           }
 ----
 <1> An array of values specifying the probability of the prediction and the 
-`class_score` for each class. The `top_classes` object contains the predicted 
-classes with the highest scores.
-<2> The probability is a value between 0 and 1. The higher the number, the more 
-confident the model is that the data point belongs to the named class. In this 
-example, `false` has a `class_probability` of 0.91 while `true` has only 0.08, 
-so the prediction will be `false`.
-<3> The `class_score` is a function of the probability. It is chosen so that the 
-decision to assign the data point to the class with the highest score maximizes 
-the minimum recall of any class.
+`class_score` for each class. 
+
+The `top_classes` object contains the predicted  classes with the highest 
+scores. The `class_probability` is a value between 0 and 1. The higher the 
+number, the more confident the model is that the data point belongs to the named 
+class. In the example above, `false` has a `class_probability` of 0.91 while 
+`true` has only 0.08, so the prediction will be `false`. The `class_score` is a 
+function of the probability. It is chosen so that the decision to assign the 
+data point to the class with the highest score maximizes the minimum recall of 
```

Comment:
> ... maximizes the minimum recall...

Have we covered recall in this example? If not, this sentence might be too advanced for this context.